src/migration/MigrationExecutor.ts

```diff
@@ -1,15 +1,25 @@
 import {Table} from "../schema-builder/table/Table";
+import {TableColumn} from "../schema-builder/table/TableColumn";
 import {Connection} from "../connection/Connection";
 import {Migration} from "./Migration";
+import {MigrationInterface} from "./MigrationInterface";
 import {ObjectLiteral} from "../common/ObjectLiteral";
+import {CollectionUtils} from "../util/CollectionUtils";
 import {PromiseUtils} from "../util/PromiseUtils";
 import {QueryRunner} from "../query-runner/QueryRunner";
 import {SqlServerDriver} from "../driver/sqlserver/SqlServerDriver";
 import {MssqlParameter} from "../driver/sqlserver/MssqlParameter";
 import {SqlServerConnectionOptions} from "../driver/sqlserver/SqlServerConnectionOptions";
 import {PostgresConnectionOptions} from "../driver/postgres/PostgresConnectionOptions";
-import { MongoDriver } from "../driver/mongodb/MongoDriver";
-import { MongoQueryRunner } from "../driver/mongodb/MongoQueryRunner";
+import {MongoDriver} from "../driver/mongodb/MongoDriver";
+import {MongoQueryRunner} from "../driver/mongodb/MongoQueryRunner";
+import {sha1} from "object-hash";
+
+export const COLUMN_CREATE_DATE = "create_date";
```

Comment:
can we remove those constants and move strings where they are used? I don't want to have in the project unknown `COLUMN_CREATE_DATE` when I search "column".

Reply:
IMHO there is so much columns in typeorm codebase. I think 5 more is not so bad (of course if it helps to readable of code). Maybe move it inside of class or call it `COLUMN_NAME_*`? Just thinking out loud.

![2r50gm](https://user-images.githubusercontent.com/2484063/51144938-bdab0f00-1852-11e9-9c5e-809bcf9a86a7.jpg)

Reply:
I'm used to extracting repeated strings into constants to prevent typos. I can revert it back to strings, or change to `COLUMN_NAME_*`, whichever you prefer.